kernelci/test.py

```diff
@@ -39,6 +39,7 @@ def match_configs(configs, bmeta, dtbs, lab):
         'kernel': bmeta['git_describe'],
         'build_environment': bmeta['build_environment'],
         'tree': bmeta['job'],
+        'branch': bmeta['git_branch']
```

Comment:
Nit: it's better to keep a trailing comma so if we add another line it keeps the diff shorter like in this patch.

Reply:
```suggestion
        'branch': bmeta['git_branch'],
```